keyboards/phantagom/varan/info.json

```diff
@@ -0,0 +1,70 @@
+{
+    "manufacturer": "Dennis Kruyt",
+    "keyboard_name": "phantagom/varan",
+    "maintainer": "dkruyt",
+    "bootloader": "rp2040",
+    "diode_direction": "COL2ROW",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "rgblight": {
+        "led_count": 23,
+        "pin": "GP28",
```

Comment:
This will need to be migrated to "ws2812.pin", once it's merged.  No action is needed, currently.